src/main/java/software/sigma/internship/properties/AllowOriginProps.java

```diff
@@ -0,0 +1,17 @@
+package software.sigma.internship.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties
+@Getter
+@Setter
+public class AllowOriginProps {
```

Comment:
`@Configuration + @EnableConfigurationProperties` could be replaced just with `@Component`